internal/environment/command_create.go

```diff
@@ -65,5 +65,5 @@ func (c *command) create(cmd *cobra.Command, args []string) error {
 
 func (c *command) addStreamGovernancePackageFlag(cmd *cobra.Command, defaultValue string) {
 	values := utils.ArrayToCommaDelimitedString([]string{"essentials", "advanced"}, "or")
-	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf("Specify the Stream Governance package as %s.", values))
+	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf("Specify the Stream Governance package as %s. Note: downgrading the package from \"advanced\" to \"essentials\" is forbidden.", values))
```

Comment:
```suggestion
	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf(`Specify the Stream Governance package as %s. Downgrading the package from "advanced" to "essentials" is not allowed.`, values))
```

Reply:
Wait, shouldn't the message about downgrading only be included in `confluent environment update`?

Reply:
I think it would be helpful for users to know before they create an environment that they can't downgrade the package, right? Users will be less likely to make an irreversible decision.

Reply:
That's a good point. Let's merge this!